packages/pn-personafisica-webapp/src/components/Contacts/__test__/PecContactItem.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { digitalLegalAddresses } from '../../../__mocks__/Contacts.mock';
+import { RenderResult, act, fireEvent, render, waitFor } from '../../../__test__/test-utils';
+import { DigitalContactsCodeVerificationProvider } from '../DigitalContactsCodeVerification.context';
+import PecContactItem from '../PecContactItem';
+
+describe('PecContactItem component', () => {
+  let result: RenderResult;
+
+  it('type in an invalid pec', async () => {
```

Comment:
add the test type in a valid pec copied from InsertLegalContact.test.tsx - checks valid pec